conflux/caip10.md

```diff
@@ -0,0 +1,68 @@
+---
+namespace-identifier: conflux-caip10
+title: Conflux Namespace - Addresses
+author: iosh(@iosh)
+status: Draft
+type: Standard
+created: 2024-08-09
+requires: ["CAIP-2", "CAIP-10"]
+---
+
+# CAIP-10
+
+_For context, see the [CAIP-10][] specification._
+
+## Rationale
+
+The Conflux Network EOA address is base32-encoded address it is derived directly from the hex-encoded addresses including a distinctive network prefix ("cfx" "cfxtest" or "net[networkId]"), an optional address type, and a checksum.
+
+The hex address is a concatenation of a 4-bit type indicator and the rightmost 156-bit Keccak digest of the associated public key of the private key.
+
+## Syntax
+
+The syntax of Conflux addresses:
+
+```
+caip10-like address:    namespace + ":" networkId + ":" + address
+namespace:              conflux
+network Id:             unsigned integer ranging from 1 to 4294967295.
+address:                conflux address represented as a base32 encoded string
+```
+
+## Test Cases
+
+The list of example address generated form the same hex address, composed use the [Address Format Conversion][] or [conflux-address-js][]
+
+```
+# hex address
+0x1a2f80341409639ea6a35bbcab8299066109aa55
+
+# mainnet
+conflux:1029:cfx:aarc9abycue0hhzgyrr53m6cxedgccrmmyybjgh4xg
+
+# testnet
+conflux:1:cfxtest:aarc9abycue0hhzgyrr53m6cxedgccrmmy8m50bu1p
+
+# private net
+conflux:2024:net2024:aarc9abycue0hhzgyrr53m6cxedgccrmmy06bvtn67
+
+# private net
+conflux:202408:net202408:aarc9abycue0hhzgyrr53m6cxedgccrmmywsh35uak
```

Comment:
CAIP-10 doesn't allow this redundant segment, and besides, you could think of the CAIP-10 format as the _machine-readable_ version of the more human-friendly and readable version, with `net202408` converted to `conflux:202408` to facilitate multi-L1 logs and wire formats.

Another question for your profile might whether to accept, for example,
`conflux:202408:0x1a2f80341409639ea6a35bbcab8299066109aa55` as an alias for
`conflux:202408:aarc9abycue0hhzgyrr53m6cxedgccrmmywsh35uak`, or vice versa. I have no opinion, just wondering, as sometimes systems with a per-chain address expression treat the "universal" underlying key differently.

Reply:
first question：The starting part of the Conflux base32 address, such as cfx, cfxtest or net[networkid], is part of the address and cannot be removed. Therefore, it is not possible to change `conflux:1029:cfx:aarc9abycue0hhzgyrr53m6cxedgccrmmyybjgh4xg` to `conflux:1029:aarc9abycue0hhzgyrr53m6cxedgccrmmyybjgh4xg`. but i think it can be changed to `conflux:cfx:aarc9abycue0hhzgyrr53m6cxedgccrmmyybjgh4xg`. the networkid because the address already contains this

Reply:
The second question:  Conflux uses a unique method to calculate EOA addresses, which means that address strings generated from the same private key will typically differ between Conflux and Ethereum. This visual similarity, combined with the computational differences, makes it easy for users to confuse Conflux addresses with Ethereum addresses, potentially leading to loss of assets. Therefore, in Conflux we do not use hex addresses.  You can find more about [base32 address](https://github.com/Conflux-Chain/CIPs/blob/master/CIPs/cip-37.md)

Reply:
First question: sure, you could use the native prefixes as the chainIds instead of the numerical ones, by just updating the CAIP-2 profile-- if `cfx` (the string) rather than `18` (the numeral) is used to calculate the checksum, that would make more sense! But the important part is the number of segments :D 

Second question: I gathered that from the links provided, but I also gathered that the "hex address" was used internally (or in legacy contexts) so I was just trying to come up with a more explanatory header than "## hex address" to reflect that.  Feel free to change for a better one, as long as it explains how it relates to the per-chain addresses derived from it!